scripts/setup-minikube-for-linux.sh

```diff
@@ -2,6 +2,10 @@
 
 set -exu
 
+# Required by minikube
+sudo apt update
+sudo apt install conntrack
```

Comment:
doing it like https://github.com/akka/akka-sample-cluster-kubernetes-java/blob/master/.travis.yml#L8 might save some time 'update'-ing, but this is :+1: as well

Reply:
Hrmm we already have that in our travis and doesn't seem to work for minikube

Reply:
Huh, so #649 worked for a while and then stopped working again? Weird.

Reply:
Current builds are now passing so i'll close this